src/utils/validators.js

```diff
@@ -15,6 +15,7 @@ import {
 const PERMALINK_REGEX = "^((/([a-zA-Z0-9]+-)*[a-zA-Z0-9]+)+)/?$"
 export const URL_REGEX_PREFIX = "^(https://)?(www.)?("
 export const URL_REGEX_SUFFIX = ".com/)([a-zA-Z0-9_-]+([/.])?)+$"
+export const YOUTUBE_REGEX = ".com/(@)?)([a-zA-Z0-9_-]+([/.])?)+$"
```

Comment:
I think we shouldn't do this because it's still possible to specify a youtube link without using alias! There's quite a few ways to specify your url, see here: https://support.google.com/youtube/answer/6180214?hl=en

Reply:
good enough as is, maybe next time can actually make regex for the 3 types of url (handle/custom/user)